app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -146,112 +146,41 @@ function ReviewIssuesStep({
-                : formatWithStrongNoWrap(
-                    t('idv.failure.attempts.other_variant_b_html', { count: remainingAttempts }),
-                  )}
-            </p>
-          )}
-          <p>{t('in_person_proofing.body.cta.prompt_detail_b')}</p>
-        </Warning>
-      );
-    }
-    if (inPersonCtaVariantActive === 'in_person_variant_c') {
-      return (
-        <Warning
-          heading={t('errors.doc_auth.throttled_heading')}
-          actionText={t('idv.failure.button.warning')}
-          actionOnClick={onWarningPageDismissed}
-          location="doc_auth_review_issues"
-          remainingAttempts={remainingAttempts}
-          troubleshootingOptions={
-            <DocumentCaptureTroubleshootingOptions
-              location="post_submission_warning"
-              showAlternativeProofingOptions={false}
-              heading={t('components.troubleshooting_options.ipp_heading')}
-            />
-          }
-        >
-          {!!unknownFieldErrors &&
-            unknownFieldErrors
-              .filter((error) => !['front', 'back'].includes(error.field!))
-              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
-
-          {remainingAttempts <= DISPLAY_ATTEMPTS && (
-            <p>
-              <strong>
-                {remainingAttempts === 1
-                  ? t('idv.failure.attempts.one')
-                  : t('idv.failure.attempts.other', { count: remainingAttempts })}
-              </strong>
-            </p>
-          )}
-        </Warning>
-      );
-    }
+    return (
+      <Warning
+        heading={t('errors.doc_auth.throttled_heading')}
+        actionText={t('idv.failure.button.warning_variant')}
+        actionOnClick={onWarningPageDismissed}
+        location="doc_auth_review_issues"
+        remainingAttempts={remainingAttempts}
+        troubleshootingOptions={
+          <DocumentCaptureTroubleshootingOptions
+            location="post_submission_warning"
+            showAlternativeProofingOptions={!isFailedResult}
+            heading={t('components.troubleshooting_options.ipp_heading')}
+            altInPersonCta={t('in_person_proofing.headings.cta_variant')}
+            altInPersonCtaButtonText={t('in_person_proofing.body.cta.button_variant')}
+          />
+        }
+      >
+        <h2>{t('errors.doc_auth.throttled_subheading')}</h2>
+        {!!unknownFieldErrors &&
+          unknownFieldErrors
+            .filter((error) => !['front', 'back'].includes(error.field!))
+            .map(({ error }) => <p key={error.message}>{error.message}</p>)}
+
+        {remainingAttempts <= DISPLAY_ATTEMPTS && (
+          <p>
+            {remainingAttempts === 1
+              ? formatWithStrongNoWrap(t('idv.failure.attempts.one_variant_a_html'))
+              : formatWithStrongNoWrap(
+                  t('idv.failure.attempts.other_variant_a_html', { count: remainingAttempts }),
+                )}
+          </p>
```

Comment:
Can we remove all references to "variant", if this is the default view?

I think this would also be a good opportunity to incorporate the suggestion I made in January, the ticket for which seems to have stagnated: https://github.com/18F/identity-idp/pull/7669#discussion_r1082753182

```suggestion
          <p>{formatWithStrongNoWrap(t('idv.failure.attempts', { count: remainingAttempts }))}</p>
```

Reply:
yes, i was thinking we should change the naming since we have 1 version of the view now. There was a question as to whether we should leave the ability to test this again in the future but I'm waiting on feedback as to where we stood on that idea. I'm thinking about just making all the changes now though as my personal opinion would be to commit to the one view we found performed better and not test again.

Reply:
Yeah, I'd also generally lean on the side of removing what we don't have a present need for, and reintroducing it later if/when we need it.

Reply:
pluralization addressed in 9407ba8951c7b053513427dc027e9a6ebede5cb7